DMX.Gatekeeper.Api.Tests.Acceptance/APIs/Homes/HomeApiTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DMX.Gatekeeper.Api.Tests.Acceptance.Brokers;
+using FluentAssertions;
+using Xunit;
+
+namespace DMX.Gatekeeper.Api.Tests.Acceptance.APIs.Homes
+{
+    [Collection(nameof(ApiTestCollection))]
+    public class HomeApiTests
+    {
+        private readonly DmxGatekeeperApiBroker dmxGatekeeperApiBroker;
+
+        public HomeApiTests(DmxGatekeeperApiBroker dmxGatekeeperApiBroker)
+        {
+            this.dmxGatekeeperApiBroker = dmxGatekeeperApiBroker;
+        }
+
+        [Fact]
+        public async Task ShouldReturnHomeMessageAsync()
+        {
+            string expectedMessage = "Hello, Goodbye";
+            string actualMessage =
```

Comment:
Separate multi-liner code with an empty line.

Here's a reference on the topic of multiple variable declarations:
https://github.com/hassanhabib/CSharpCodingStandard/blob/master/Variables.md#021-multiple-declarations